browser/ui/webui/brave_webui_source.cc

```diff
@@ -364,6 +366,14 @@ void CustomizeWebUIHTMLSource(const std::string &name,
         { "expiresOn", IDS_BRAVE_UI_EXPIRES_ON },
         { "for", IDS_BRAVE_UI_FOR },
         { "grantDisclaimer", IDS_BRAVE_UI_GRANT_DISCLAIMER },
+        { "grantTitleUGP", IDS_BRAVE_UI_GRANT_TITLE_UGP },
```

Comment:
where is this used?

Reply:
`components/resources/brave_components_strings.grd:      <message name="IDS_BRAVE_UI_GRANT_TITLE_UGP" desc="Title for a reward grant when it has been claimed">It's your lucky day!</message>`

Reply:
@ryanml you added it a while ago. Is it used today?

Reply:
@samartnik I see it in use here: https://github.com/brave/browser-android-tabs/blob/base-76.0.3809.132/components/brave_rewards_ui/resources/components/grant.tsx#L52

Reply:
@ryanml how come that we are not using the same strings as on desktop?

Reply:
@ryanml @NejcZdovc as far as I understand we need it. Can we resolve this conversation?

Reply:
@samartnik please log an issue on brave core and android for this and assign @ryanml to them, so that we remove duplication.

Reply:
Issues created https://github.com/brave/brave-browser/issues/6302
I don't have permissions to assign it to @ryanml

Reply:
done